src/core/link-to-dfn.js

```diff
@@ -287,17 +287,16 @@ function showLinkingError(elems) {
  * @param {Conf} conf
  */
 function updateReferences(conf) {
-  const shortName = new RegExp(
-    String.raw`\b${(conf.shortName || "").toLowerCase()}([^-])\b`,
-    "i"
-  );
+  const { shortName = "" } = conf;
+  // https://regex101.com/r/rsZyIJ/5
```

Comment:
Prefer we leave a comment, as regex101 could vanish.

Reply:
I did archive it 🤺